app/src/main/java/org/fossasia/pslab/adapters/PerformExperimentAdapter.java

```diff
@@ -67,93 +67,93 @@ public Fragment getItem(int position) {
                     case "Zener IV Characteristics":
                         return ExperimentDocFragment.newInstance("D_ZenerIV.html");
```

Comment:
This has a md file :)

Reply:
Fixed :+1:

Reply:
Awesome! There's a lot of files. So can miss something!